src/main/java/io/cdap/plugin/gcp/bigquery/sink/AbstractBigQuerySink.java

```diff
@@ -386,6 +392,11 @@ protected void validateSchema(Table table, Schema tableSchema, boolean allowSche
     collector.getOrThrowException();
   }
 
+  private Boolean isTruncateTableSet() {
+    AbstractBigQuerySinkConfig config = getConfig();
+    return (config == null || config.truncateTable == null) ? false : config.truncateTable;
```

Comment:
config will never be null, this method can be replaced with a call to config's isTruncateTableSet

Reply:
Done